fixtures/cassettes/orders_responses.yml

```diff
@@ -121,6 +121,68 @@ d0c2a1b2a0fd3078fcb7492bc2ea067cd82834e8:
       encoding: UTF-8
       string: " "
     http_version: 
+ae82d2a1dc8801714b10b5ddc4152532641556d8:
+  response:
+    status:
+      code: 404
```

Comment:
They are not intentional but won't affect tests either. There are a couple of other existing appearances of 404 error with this order GUID (d0c2a1b2a0fd3078fcb7492bc2ea067cd82834e8 is one of them).

The tests are not affected because they don't assert on these responses.